handlers/deploy.go

```diff
@@ -136,6 +136,13 @@ func makeDeploymentSpec(request requests.CreateFunctionRequest) *v1beta1.Deploym
 								},
 							},
 							ImagePullPolicy: v1.PullAlways,
+							LivenessProbe: &apiv1.Probe{
+								Handler: apiv1.Handler{
+									Exec: &apiv1.ExecAction{
```

Comment:
I assume this creates some defaults for time until first run and time between repeating the check? For maintainability we should create the values explicitly

Reply:
I set the value for the initial delay to 10, but couldn't figure out what the default value is.